src/fake.ts

```diff
@@ -15,25 +15,35 @@ export class Fake {
   }
 
   /**
-   * Generator method for combining faker methods based on string input.
+   * Generator for combining faker methods based on a static string input.
    *
-   * Note: If you just want to create a string on the fly, we recommend using string template literals instead.
-   * This method is useful if you wish to choose a random format from a non-executable source or persistent storage (json etc.).
+   * Note: We recommend using string template literals instead of `fake`,
+   * which are faster and strongly typed (if you are using TypeScript),
+   * e.g. `const address = `${faker.address.zipCode} ${faker.address.city}``
```

Comment:
```suggestion
   * e.g. ``const address = `${faker.address.zipCode} ${faker.address.city}`;``
```

You have to use two `` ` `` because you use a single `` ` `` inside.